modules/GKB/Release/Step.pm

```diff
@@ -418,11 +418,11 @@ sub archive_files {
 
     system "mkdir -p $step_version_archive";
     if (-d $step_version_archive) {
-        foreach my $sql_dump_file (glob '*.dump') {
+        foreach my $sql_dump_file (glob "*.dump") {
             if (-e "$sql_dump_file.gz") {
                 system "mv --backup=numbered $sql_dump_file.gz $step_version_archive";
             }
-            system "gzip -qf $step_version_archive/$sql_dump_file 2> /dev/null";
+            system "gzip -qf $sql_dump_file";
```

Comment:
Why was `$step_version_archive/` removed from the path to the SQL dump file?

Reply:
This was a bug - the zipping happens before moving to the `$step_version_archive` directory, so it was trying to zip in the wrong directory and producing a 'file not found' error.  It needs to zip the dump files in the current directory then move them to `$step_version_archive`.

Reply:
Ah ok.